### ISSUE TITLE
pretrain the OSNet in ImageNet

### ISSUE THREAD

**Issue by hjiang97321** [KaiyangZhou/deep-person-reid#247]
I want to modify some structures of the OSNet, can you give me some advice about how to pretrain the model in ImageNet?

**Comment by KaiyangZhou** [comment]
you can use this code https://github.com/switchablenorms/Switchable-Normalization/tree/master/imagenet to do the imagenet training

**Comment by hjiang97321** [comment]
OK, thank you very much!

**Comment by valuefish** [comment]
I want to reproduce the experiment of pretraining OSNet with ImageNet, are the hyperparameters same as [Switchable-Normalization/imagenet/configs/config_resnetv1sn50_cosine.yaml
](https://github.com/switchablenorms/Switchable-Normalization/blob/master/imagenet/configs/config_resnetv1sn50_cosine.yaml)?

**Comment by yuanry16** [comment]
I also want to reproduce the experiment of pretraining OSNet with ImageNet. My ques is  the stride list is [2,2,2,2,2] 32 downsample or [2,2,2,2,1] 16 downsample in the imagenet cls task stage?
